textalyze.rb

```diff
@@ -52,22 +52,46 @@ def read_file(filename)
   begin
     file_contents = File.read(filename)
   rescue
-    puts "Error opening filename"
+    puts "Error opening file"
   end
   return file_contents
 end
 
+def freq_stat(hash)
+  total = 0.0
+  freq_hash = Hash.new(0)
+  hash.each do |k, v|
```

Comment:
Variable names like `hash`, `k`, and `v` are very hard to reason about. What is the meaning of `k` to a human? Is it a word? A character? A spaceship?